pelican/tests/build_test/test_wheel.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+import pytest
+from zipfile import ZipFile
+
+
+@pytest.mark.skipif(
+    "not config.getoption('--check-wheel')",
+    reason="Only run when --check-wheel is given",
+)
+def test_wheel_contents(pytestconfig):
+    """
+    This test, should test the contents of the wheel to make sure,
+    that everything that is needed is included in the final build
```

Comment:
I suggest removing the two commas in this sentence, as well as adding a period at the end.